javascript/javascript1/week4/classexercise.js

```diff
@@ -0,0 +1,15 @@
+function printNum(num1, num2) {
```

Comment:
this is the correct solution but I don't see any use of `num1` & `num2` in your function, so is it necessary to have them??